src/main/java/com/iexec/common/docker/client/DockerClientInstance.java

```diff
@@ -261,17 +262,29 @@ public synchronized boolean removeNetwork(String networkName) {
 
     /**
      * Docker image
-     * 
+     *
      */
 
     /**
      * Pull docker image and timeout after 1 minute.
      * 
-     * @param imageName
+     * @param imageName Name of the image to pull
      * @return true if image is pulled successfully,
      * false otherwise.
      */
     public boolean pullImage(String imageName) {
```

Comment:
If we keep this method we can add the duration of 1 minute in javadoc.

Reply:
Where would you like me to add it? It is already in `pullImage` JavaDoc:
```
Pull docker image and timeout after 1 minute.
```

Reply:
Oh I totally missed, my bad!